### DISCUSSION TITLE
New maintainer(s) wanted

### DISCUSSION THREAD

**Discussion by djwatson** [libunwind/libunwind#627] [Announcements]
Unfortunately after ~5 years I've almost run out of bandwidth for this project.  If you're interested in helping or taking over maintenance, email or leave a comment.  Thanks.

Currently I'll continue until at least the next release (v1.7 probably in about jan 2023)

**Comment by StefanKarpinski** [comment]
@djwatson, as grateful long-time users of libunwind, the Julia project would like to treat you to a spa appointment so that you can... unwind 😁

Thank you for all your hard work. DM me [on Twitter](https://twitter.com/StefanKarpinski) and I'll send you a gift card.
